fastcrypto/src/ed25519.rs

```diff
@@ -444,6 +444,26 @@ impl AggregateAuthenticator for Ed25519AggregateSignature {
             .map_err(|_| FastCryptoError::GeneralError)
     }
 
+    fn verify_different_msg(
```

Comment:
what's the difference between this and `batch_verify`?

Reply:
As I understand it, `batch_verify`verifies a batch of aggregate signatures, where each aggregate signature is on the same message, but `verify_different_msg`verifies a single aggregate signature different messages.

Reply:
Ok will approve now and we can work on the ideal api later.

Reply:
Great. To keep consistency, we might also want to add `batch_verify_different_msg`?